### ISSUE TITLE
experienceUpdate

### ISSUE THREAD

**Issue by Salvationdk** [pulviscriptor/agario-client#131]
is this packet changed also ?

- `on.experienceUpdate(level, current_exp, need_exp)` experience information update (if logined with [auth token](#auth-token))

cant seem to get it to retrieve info

**Comment by SzAmmi** [comment]
Nope, it works but the login packet changed so this is why u didn't get `experienceUpdate`.

**Comment by Salvationdk** [comment]
already got the login change...

**Comment by SrNicolas** [comment]
@SzAmmi Hey, I need to contact you on skype, please add me on skype: nicolasdelito

**Comment by pulviscriptor** [comment]
I planning to remove `experienceUpdate` event since i think its now in miniclip server (which is not supported by agario-client).
Does `experienceUpdate` works for anybody? Can i remove it?

**Comment by SrNicolas** [comment]
not working for me, remove it
